### ISSUE TITLE
Pylons Language Updates

### ISSUE THREAD

**Issue by heyitsjk** [Pylons-tech/pylons#878]
## Summary

Update English i18n in Pylons for some strings (grammar/formatting corrections) + Add missing keys to all other languages.

- Also addresses some i18n corrections in #871 + Attempted formatting fix for hyphenated letters. 

New: line numbers all now match the EN file. So finding a key in one language will be on the same line in other i18n files. 

[de.txt](https://github.com/Pylons-tech/pylons/files/9422999/de.txt)
[en.txt](https://github.com/Pylons-tech/pylons/files/9423000/en.txt)
[es.txt](https://github.com/Pylons-tech/pylons/files/9423001/es.txt)
[id.txt](https://github.com/Pylons-tech/pylons/files/9423002/id.txt)
[ja.txt](https://github.com/Pylons-tech/pylons/files/9423003/ja.txt)
[ko.txt](https://github.com/Pylons-tech/pylons/files/9423004/ko.txt)
[ru.txt](https://github.com/Pylons-tech/pylons/files/9423005/ru.txt)
[vi.txt](https://github.com/Pylons-tech/pylons/files/9423007/vi.txt)

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by faizhashmiRnS** [comment]
@kjawadDeveloper2 should we replace these i18n files mentioned above with the files in Wallet?

**Comment by kjawadDeveloper2** [comment]
i will do it .

**Comment by heyitsjk** [comment]
> @kjawadDeveloper2 should we replace these i18n files mentioned above with the files in Wallet?

Just paste whats in each .txt into the corresponding .json file - as GitHub won't let me upload .json files to these reports

**Comment by kjawadDeveloper2** [comment]
@heyitsjk  what about this 
Before: Pylonen Geldbörse
After: Pylonen-Geldbörse

**Comment by heyitsjk** [comment]
@kjawadDeveloper2 lets use "Pylons Geldbörse"

**Comment by kjawadDeveloper2** [comment]
@mustafapylons as per discussion with @heyitsjk  will again look into translations.

**Comment by heyitsjk** [comment]
Pulled from yesterdays i18n fixes -- these should all be good to go now @kjawadDeveloper2 

[de.txt](https://github.com/Pylons-tech/pylons/files/9435124/de.txt)
[en.txt](https://github.com/Pylons-tech/pylons/files/9435126/en.txt)
[es.txt](https://github.com/Pylons-tech/pylons/files/9435127/es.txt)
[id.txt](https://github.com/Pylons-tech/pylons/files/9435128/id.txt)
[ja.txt](https://github.com/Pylons-tech/pylons/files/9435129/ja.txt)
[ko.txt](https://github.com/Pylons-tech/pylons/files/9435130/ko.txt)
[ru.txt](https://github.com/Pylons-tech/pylons/files/9435131/ru.txt)
[vi.txt](https://github.com/Pylons-tech/pylons/files/9435132/vi.txt)

**Comment by StefanB90** [comment]
Low priority. Please check when possible to close this issue. Thank you @faizhashmiRnS

**Comment by faizhashmiRnS** [comment]
@kjawadDeveloper2 are they replaced in the code? I find these similar with the ones in code.

**Comment by kjawadDeveloper2** [comment]
Yes @Tanyaml  kindly close this ticket.
